diff --git a/index.d.ts b/index.d.ts
index aee3041de0bc2a46df7bcfca387152bf2cc01e30..73b147691d00fa5ee4e926fdf8ef712ab226a9a3 100644
--- a/index.d.ts
+++ b/index.d.ts
@@ -757,6 +757,9 @@ export interface Response<
      */
     json: Send<ResBody, this>;
 
+    success: (data?: any) => void;
+    fail: (message: string, code?: number) => void;
+    
     /**
      * Send JSON response with JSONP callback support.
      *
